Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs

```diff
@@ -35,32 +39,49 @@ namespace Trisoft.ISHRemote.Cmdlets.BackgroundTask
     /// </summary>
```

Comment:
Do we still need to mention DocumentObj25.RaiseEventByIshLngRefs in the description?

Reply:
I removed it and corrected the whole description.